hail_scripts/v01/utils/computed_fields.py

```diff
@@ -26,8 +26,6 @@
     "NMD_transcript_variant",
     "non_coding_transcript_variant",
     "nc_transcript_variant",  # deprecated
-    "upstream_gene_variant",
-    "downstream_gene_variant",
```

Comment:
This breaks https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/92700c1b494fd702487a3f52c2bc4a3a9f1d332d/download_and_create_reference_datasets/v01/hail_scripts/write_dataset_validation_kt.py#L71-L72

Reply:
Hmm could we use intronic variants instead?

Reply:
Either
```
... major_consequence_rank.toInt > {}.get("nc_transcript_variant").toInt ...
```
or
```
... major_consequence_rank.toInt >= {}.get("TFBS_ablation").toInt ...
```
would preserve the same behavior as before.

But I don't understand why when splitting into coding and non-coding variants, the 11 VEP consequences between synonymous_variant and downstream_gene_variant are omitted.

Reply:
I think @bw2 wrote this and might have some insight into that